data/human/outfits.txt

```diff
@@ -409,14 +409,14 @@ outfit "Navy License"
 	thumbnail "outfit/navy license"
 	description "It is unusual, but not unheard of, for private citizens to be given the right to purchase Navy ships for their own private use. Recently, certain rich worlds have also begun petitioning Parliament for special permission to purchase Navy warships for their private security forces."
 
-outfit "Cruiser License"
+outfit "Navy Cruiser License"
 	category "Special"
-	thumbnail "outfit/cruiser license"
+	thumbnail "outfit/navy cruiser license"
 	description "It is a very rare privilege to be permitted to purchase a Navy capital ship for private use - not just because of a Cruiser's firepower, but because it is a potent symbol of the Republic's strength."
 
-outfit "Carrier License"
+outfit "Navy Carrier License"
 	category "Special"
-	thumbnail "outfit/carrier license"
+	thumbnail "outfit/navy carrier license"
```

Comment:
If we didn't change the image name then we wouldn't need a high-DPI plugin as well, but given that it's already opened, little point in going back now I guess.

Reply:
OK.